content/en/docs/concepts/workloads/autoscaling.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Autoscaling Workloads
+description: >-
+  With autoscaling, you can automatically update your workloads in one way or another. This allows your cluster to react to changes in resource demand more elastically and efficiently.
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+In Kubernetes, you can _scale_ a workload depending on the current demand of resources.
+This allows your cluster to react to changes in resource demand more elastically and efficiently.
+
+There are manual and automatic ways to scale your workloads, depending on your use case.
+
+<!-- body -->
+
+## Scaling workloads manually
+
+Kubernetes supports _manual scaling_ of workloads, either by changing the number of
+{{< glossary_tooltip text="Replicas" term_id="replica">}} of an object that manages a set of
+{{< glossary_tooltip text="Pods" term_id="pod" >}} (for example a {{< glossary_tooltip text="Deployment" term_id="deployment" >}}),
+or by adjusting the provided resources of each Replica (for example CPU or memory).
+
+### Scaling the number of Replicas of a workload
```

Comment:
```suggestion
### Manual changes to replica count
```

As this is a a concept pages, you're defining concepts (like the idea that you can manually change replica count).
Separately, we could make a task page about this - perhaps under https://kubernetes.io/docs/tasks/run-application/ - that reiterates how to run `kubectl scale`.

Reply:
I need some clarification on this - do you want me to not go into detail regarding how to (manually) scale workloads at all, or do you want me to adjust the section headers to make it sound less of a tutorial, but keep the short paragraphs below?

For these paragraphs, I took a look at the [Deployment concept page](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) which also gives short outlines of e.g. how to create deployments using a manifest.